index.html

```diff
@@ -466,6 +466,12 @@ <h1>Terminology</h1>
                 Mozilla Project Things with its own specifications for
                 application-facing APIs, data model, and protocols or
                 protocol configurations.</dd>
+            <dt>
+                <dfn>Metadata</dfn>
+            </dt>
+            <dd>Data that provides a description of an entity's abstract characteristics or situtation
```

Comment:
suggest to just use:
Data that provides a description of an entity's abstract characteristics.

Reply:
fixed.